rdoc.gemspec

```
@@ -37,7 +37,7 @@ RDoc includes the +rdoc+ and +ri+ tools for generating and displaying documentat
   s.require_paths = ["lib"]
   # for ruby core repository. It was generated by
   # `git ls-files -z`.split("\x0").each {|f| puts "    #{f.dump}," unless f.start_with?(*%W[test/ spec/ features/ .]) }
-  s.files = [
+  NONE_LIB_FILES = [
```

Non-`lib/` files, perhaps.

```suggestion
  NON_LIB_FILES = [
```